### PR TITLE
1281 description of server error

### PR DESCRIPTION
**Pull request by MalikJuly**

### CODE REVIEW

File: VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeNavigation.js [hunk 1]
```diff
@@ -191,8 +191,16 @@ angular.module('platformWebApp')
                 event.stopPropagation();
                 $document.bind('click', handleClickEvent);
             };
+
+            scope.seeDetailsLabel = "see Details";
+            scope.showErrorDetails = function () {
+                var dialog = { id: "errorDetails", title: "Description of error." };
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeNavigation.js]: Description of dialog should get title and labels from localized resources

**Comment by MalikJuly** [comment]
error-screen

File: VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeContainer.tpl.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <div class="blade-wrapper" va-blade='blade'>
         <header class="blade-head">
             <div class="blade-status" ng-class="{'__error': blade.error}">
-                <div class="status-message" ng-if="blade.error">{{blade.error}}</div>
+                <div class="status-message" style="width: 76%;" ng-if="blade.error"><div>{{blade.error}}<a class="link-error-details" style="margin-right: 5px;" ng-click="showErrorDetails()">{{ 'platform.commands.see-details' | translate }}</a></div></div>
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeContainer.tpl.html]: need to get rid these inline styles

File: VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeNavigation.js [hunk 3]
```diff
@@ -392,10 +399,12 @@ angular.module('platformWebApp')
         setError: function (msg, blade) {
             if (blade) {
                 blade.isLoading = false;
-                blade.error = msg;
+                blade.error = $.trim(msg);
+                if (blade.error.length > 55)
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeNavigation.js]: Such a logic can be performed using css styles, the controller logic should ignore the details of the presentation

**Comment by MalikJuly** [comment]
error-screen2

**Pull request merged**